### ISSUE TITLE
Pub/Sub RPC feature

### ISSUE THREAD

**Issue by tymat** [paritytech/parity#884]
See: https://github.com/ethereum/go-ethereum/wiki/RPC-PUB-SUB

**Comment by gavofyork** [comment]
despite the go guys insistence, full duplex operations is not within the JSONRPC v2 standard. that said, we might support it if it's not too much effort.

**Comment by rphmeier** [comment]
There seems to be quite a bit of demand for this feature. :+1: to get it in for 1.4

**Comment by jacogr** [comment]
We do support a subset of this (obviously implemented via polling, not via events from Parity) in our own JS API. So we need to align the 2 sides when doing this in parallel - in addition, need to carefully look at the actual use-case and not (imho) just do a straight as-is implementation.

**Comment by gituser** [comment]
would be nice to have a transaction notification feature on all wallets stored in the client like in bitcoin.
cheers

**Comment by gituser** [comment]
Guess this has been postponed. Would be nice to have feature analog of [bitcoin's walletnotify](http://bitcoin.stackexchange.com/questions/24457/how-do-i-use-walletnotify) where you can pass incoming txid to any of the wallet addresses to the external script or notify backend instead of querying parity every minute.

**Comment by 5chdn** [comment]
This issue is labelled with _sometimesoon_: Issue is worth doing soon. But it is inactive for several months.

It was initially scheduled for 1.4, now it's attached to the 1.7 milestone. Is this a realistic deadline?

**Comment by tomusdrw** [comment]
Yup, initial part ready in #5456 we still may want to implement geth-compatible subscriptions (I'll work on that soon)

**Comment by 5chdn** [comment]
Is eth_pubsub complete now?

**Comment by tomusdrw** [comment]
Still missing less important methods, I'll implement them next week to finally close this:
- [ ] syncing
- [ ] Pending (local?) transactions

**Comment by debris** [comment]
@tomusdrw what is the status of it?

**Comment by tomusdrw** [comment]
Still missing those two methods, but on my list.

**Comment by tomusdrw** [comment]
It's actually tracked in #6426, let's close this one.
